src/fides/api/ctl/migrations/versions/6d6b0b7cbb36_add_privacynotice_tables.py

```diff
@@ -0,0 +1,122 @@
+"""add privacynotice tables
+
+Revision ID: 6d6b0b7cbb36
+Revises: 8615ac51e78b
+Create Date: 2023-03-22 12:09:47.395190
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "6d6b0b7cbb36"
+down_revision = "8615ac51e78b"
```

Comment:
hmm no i think this is correct, i had to update this morning due to the migration added [here](https://github.com/ethyca/fides/pull/2906/files#diff-965f0fa50feac3d6510aa833a7570a9e50a5bc68ee8c81af4a2137ad086c0980) -- let me know if that still looks off to you @pattisdr  :)

(tests were indeed failing before i pushed up this change earlier)

Reply:
But that migration is not on this branch @adamsachs - you should get the branch updated with main then.  Checking this branch out and testing in isolation you can't even bring up the application because of it.

Reply:
oh, good point. i think it _would_ be OK after merging (since that's what the tests are evaluating), but you're right, definitely cleaner to get up to date with `main`. thanks for pointing out.

Reply:
OK, this should be good now that i've rebased. thanks for the diligence @pattisdr , definitely can feel better about merging with this rebased off latest in `main`!